src/client/App.js

```diff
@@ -33,6 +34,9 @@ function App() {
           >
             <Profile />
           </AuthenticatedRoute>
+          <Route>
+            <Overview exact path="/overview" />
+          </Route>
```

Comment:
This is where the profile routing should be set up. BUT it is not part of your task, so dont do it